### DISCUSSION TITLE
Please provide 2.4.1 version

### DISCUSSION THREAD

**Discussion by k8kiss** [lllyasviel/Fooocus#3100] [Q&A] [answered]
Does anyone has the 2.41. version? 
In the 2.4.2 the tcd sampler breaks the image generation. I've been using this sampler with good results. If anyone has the 2.4.1 files.I can't find them in the older versions rep.

Thanks in advance!

**Comment by ILikeToasters** [comment]
Here: https://github.com/lllyasviel/Fooocus/releases

**Comment by mashb1t** [comment, marked as answer]
@k8kiss relates to issue https://github.com/lllyasviel/Fooocus/issues/3099. You can always revert by using `git checkout tags/v2.4.1` and changing the entrypoint to launch.py instead of entry_with_update.py.
